rspec-rails.gemspec

```diff
@@ -27,9 +27,9 @@ Gem::Specification.new do |s|
     s.cert_chain = [File.expand_path('~/.gem/rspec-gem-public_cert.pem')]
   end
 
-  s.add_runtime_dependency(%q<activesupport>, [">= 3.0"])
-  s.add_runtime_dependency(%q<actionpack>, [">= 3.0"])
-  s.add_runtime_dependency(%q<railties>, [">= 3.0"])
+  s.add_runtime_dependency(%q<activesupport>, [">= 3.0", "<= 4.2"])
+  s.add_runtime_dependency(%q<actionpack>, [">= 3.0", "<= 4.2"])
+  s.add_runtime_dependency(%q<railties>, [">= 3.0", "<= 4.2"])
```

Comment:
While the rails team has publicly announced that 4.2 will be the last 4.x release, wouldn't it be slightly safer to do `< 5.0`?  If for some reason they decided to release 4.3 that would allow that version.  Unlikely, I know, but if your intention here is to explicitly not support rails 5+ that seems more semantically correct.  (OTOH, if you explicitly would not want to support 4.3 if that was ever released, this is fine).